### PR TITLE
Adding package idiap-devtools

### PR DESCRIPTION
**Pull request by anjos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by anjos** [comment]
@conda-forge/help-python please consider this recipe.

**Comment by anjos** [comment]
Please note this is a unix-only package - I added `__unix` to the run requirements of the package to ensure this.

Other than that, both Linux and MacOSX targets are working fine!  Thanks for considering!

File: recipes/idiap-devtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/idiap-devtools-{{ version }}.tar.gz
+  sha256: 97d8b0b1b37ff808d0c24c27069bb64aa16eed2f30e91e2cb18d7d88a2252012
+
+build:
```
Comment (cav71) [inline recipes/idiap-devtools/meta.yaml:12]: Please add this to disable win builds:
```
build:
  skip: True  # [win]
```

File: recipes/idiap-devtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - packaging
+    - python-dateutil
+    - python-gitlab
+    - pytz
+    - pyyaml
+    - setuptools
+    - tomli
+    - tomlkit
+    - xdg
+    - __unix
```
Comment (cav71) [inline recipes/idiap-devtools/meta.yaml:37]: I'm not sure this is needed (the skip build above should fix that):
```
  - __unix
```

Comment (anjos) [reply recipes/idiap-devtools/meta.yaml:37]: Thanks for your review.  Please note this is a `noarch: python` package for unix-only (macOS or Linux).  However, as conda-forge is setup, noarch packages are still installable on Windows environments.  This runtime requirement guarantees this package won't be installable on Windows.  It was recommended to me by @beckermr.

[the remark this answers is not in the document]
Comment (anjos) [reply recipes/idiap-devtools/meta.yaml:37]: It is explained above.

File: recipes/idiap-devtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/idiap-devtools-{{ version }}.tar.gz
+  sha256: 97d8b0b1b37ff808d0c24c27069bb64aa16eed2f30e91e2cb18d7d88a2252012
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
[the remark this answers is not in the document]
Comment (anjos) [reply recipes/idiap-devtools/meta.yaml:15]: Thanks for the suggestions, @marcelotrevisani.  This PR was merged by @beckermr about 6 hours ago.  Can you, instead, create a PR for the already existing package at ../idiap-devtools-feedstock?  Thanks in advance!

File: recipes/idiap-devtools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    - __unix
+
+test:
+  source_files:
+    - tests
+  imports:
+    - idiap_devtools
+  commands:
+    - pip check
+    - pytest -sv tests
```
[the remark this answers is not in the document]
Comment (anjos) [reply recipes/idiap-devtools/meta.yaml:46]: Also useful, can you please move all this to a PR? Thx!

[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/idiap-devtools/meta.yaml:46]: I forgot to press the "submit review" on gh, my bad
when I saw that I forgot to do it, it was late 😄

**Pull request merged**